docs/Usage/Cells.md

```diff
@@ -0,0 +1,100 @@
+ # Cells
+ 
+Cells are of three types − Code, Markdown and Raw.
+
+* #### Code Cells
+Contents in this cell are treated as statements in a programming language of current [kernel](../Installation.md#kernels). When a cell is run (See [Hydrogen: Run Cell](GettingStarted.md#hydrogen-run-cell)), its result is displayed inline. 
```

Comment:
"A code cell allows you to edit and write new code. The programming language you use depends on the [**Kernel**](../Installation.md#kernels) you chose to use.

When the cell is run, the text is sent as code to the file's kernel, which will run the code. The results that are returned are then displayed inline or optionally in the **Output Area**. Output however could be of many forms not just text. For example, you can output [graphs/plots](#link-to-example) or [HTML](#link-to-example)."

Reply:
@wadethestealth 
> For example, you can output graphs/plots or HTML."

For the graphs/plots and HTML example, do you prefer I add the code/screenshot in the Code Cells Example section or a link to an existing code/screenshot in the Documentation(Eg. Plot output example in the Getting Started page